src/applications/simple-forms/21-0845/containers/IntroductionPage.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @department-of-veterans-affairs/prefer-telephone-component */
+// <va-telephone /> doesn't display 1-800 numbers correctly
```

Comment:
If this is an issue with the va-telephone web component, can you create a support ticket for the design system team so that it can be fixed? 🙏🏻

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new?assignees=&labels=vsp-design-system-team%2C+bug&projects=&template=1_bug_template.md&title=

Reply:
@tlei123 and @rhasselle-oddball - To echo Jami here, please don't disable linting without at least filing an issue. We really need to deprecate usage of React components and Telephone is the most prevalent. Also, we've been, and will continue, to prioritize your requests. Thus we may be able to turn around a fix for you quickly. Thanks.

Reply:
@jamigibbs , thanks for reaching out!  And yes, I'll open a bug ticket as requested.